src/components/Button.tsx

```diff
@@ -6,12 +6,12 @@ export class Button extends React.Component<{
   title?: string;
   isDisabled?: boolean;
   onClick?: Function;
-  customClass?: string;
+  customClassName?: string;
 }, {}> {
   render() {
     let className = "button";
-    if(this.props.customClass){
-        className = this.props.customClass
+    if(this.props.customClassName){
+      className += this.props.customClassName
```

Comment:
It's better to insert the space here `className += " " + this.props.customClassName` than rely on doing so at each call site. Also, a missing semicolon.